packages/phishing-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/phishing-controller",
-  "version": "12.2.0",
+  "version": "12.3.0",
```

Comment:
According to SemVer, the minor part of the version should be bumped if there is new functionality being introduced. Is this the case? Otherwise, would bumping the patch part communicate the changes better?

``` suggestion
  "version": "12.2.1",

Reply:
I reviewed the PR. Disregard! I think we are good here.